### PR TITLE
update docs related to fixes with CMK

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-aura-338.surge.sh

File: modules/ROOT/partials/import-database.adoc [hunk 1]
```diff
@@ -32,7 +32,10 @@ For details of how to use the `neo4j-admin database upload` command, along with
 
 [NOTE]
 ====
-The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j Admin version 4.4 and earlier. If you have an earlier version of the Neo4j Admin tool, please see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
+The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j Admin version 4.4 and earlier. 
+If you have an earlier version of the Neo4j Admin tool, please see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
+
+The `neo4j-admin push-to-cloud` command in Neo4j Admin version 4.4 and earlier is not compatible with instances encrypted with link:{neo4j-docs-base-uri}/aura/platform/connectors/security[Customer Managed Keys.] Please use `neo4j-admin database upload` in Neo4j Admin version 5.x to upload data to instances encrypted with Customer Managed Keys.
```
Comment (AlexicaWright) [inline modules/ROOT/partials/import-database.adoc:38]:
```suggestion
If you have an earlier version of the Neo4j Admin tool, see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].

The `neo4j-admin push-to-cloud` command in Neo4j Admin version 4.4 and earlier is not compatible with instances encrypted with link:{neo4j-docs-base-uri}/aura/platform/connectors/security[Customer Managed Keys]. 
Use `neo4j-admin database upload` in Neo4j Admin version 5.x to upload data to instances encrypted with Customer Managed Keys.
```

Comment (AlexicaWright) [reply modules/ROOT/partials/import-database.adoc:38]: We try to avoid the use of _please_.

File: modules/ROOT/partials/import-database.adoc [hunk 2]
```diff
@@ -32,7 +32,11 @@ For details of how to use the `neo4j-admin database upload` command, along with
 
 [NOTE]
 ====
-The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j Admin version 4.4 and earlier. If you have an earlier version of the Neo4j Admin tool, please see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
+The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j Admin version 4.4 and earlier. 
+If you have an earlier version of the Neo4j Admin tool, see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
```
Comment (nvitucci) [inline modules/ROOT/partials/import-database.adoc:36]: I would skip this sentence and put the link in the previous one.

File: modules/ROOT/partials/import-database.adoc [hunk 3]
```diff
@@ -32,7 +32,11 @@ For details of how to use the `neo4j-admin database upload` command, along with
 
 [NOTE]
 ====
-The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j Admin version 4.4 and earlier. If you have an earlier version of the Neo4j Admin tool, please see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
+The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j Admin version 4.4 and earlier.
```
Comment (AlexicaWright) [inline modules/ROOT/partials/import-database.adoc:35]:
```suggestion
The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j v4.4 and earlier. 
```

File: modules/ROOT/partials/import-database.adoc [hunk 4]
```diff
@@ -32,7 +32,11 @@ For details of how to use the `neo4j-admin database upload` command, along with
 
 [NOTE]
 ====
-The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j Admin version 4.4 and earlier. If you have an earlier version of the Neo4j Admin tool, please see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
+The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j Admin version 4.4 and earlier. 
+If you have an earlier version of the Neo4j Admin tool, see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
+
+The `neo4j-admin push-to-cloud` command in Neo4j Admin version 4.4 and earlier is not compatible with instances encrypted with link:{neo4j-docs-base-uri}/aura/platform/connectors/security[Customer Managed Keys].
```
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply modules/ROOT/partials/import-database.adoc:38]: Also, it's not the neo4j-admin tool that is versioned, it's Neo4j. 
```suggestion
The `neo4j-admin push-to-cloud` command in Neo4j v4.4 and earlier is not compatible with instances encrypted with link:{neo4j-docs-base-uri}/aura/platform/connectors/security[Customer Managed Keys]. 
```

File: modules/ROOT/partials/import-database.adoc [hunk 5]
```diff
@@ -32,7 +32,11 @@ For details of how to use the `neo4j-admin database upload` command, along with
 
 [NOTE]
 ====
-The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j Admin version 4.4 and earlier. If you have an earlier version of the Neo4j Admin tool, please see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
+The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j Admin version 4.4 and earlier. 
+If you have an earlier version of the Neo4j Admin tool, see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
+
+The `neo4j-admin push-to-cloud` command in Neo4j Admin version 4.4 and earlier is not compatible with instances encrypted with link:{neo4j-docs-base-uri}/aura/platform/connectors/security[Customer Managed Keys]. 
+Use `neo4j-admin database upload` in Neo4j Admin version 5.x to upload data to instances encrypted with Customer Managed Keys.
```
Comment (AlexicaWright) [inline modules/ROOT/partials/import-database.adoc:39]:
```suggestion
Use `neo4j-admin database upload` in Neo4j 5 to upload data to instances encrypted with Customer Managed Keys.
```

**Review by AlexicaWright — COMMENTED** [review]
Some editorial suggestions.

File: modules/ROOT/partials/import-database.adoc [hunk 6]
```diff
@@ -32,7 +32,11 @@ For details of how to use the `neo4j-admin database upload` command, along with
 
 [NOTE]
 ====
-The `database upload` command was introduced in Neo4j Admin version 5, replacing the `push-to-cloud` command that was present in Neo4j Admin version 4.4 and earlier. If you have an earlier version of the Neo4j Admin tool, please see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
+The `database upload` command was introduced in Neo4j 5, replacing the `push-to-cloud` command that was present in Neo4j v4.4 and earlier. 
+If you have an earlier version of the Neo4j Admin tool, see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
```
Comment (AlexicaWright) [inline modules/ROOT/partials/import-database.adoc:36]:
```suggestion
If you have an earlier version of Neo4j, see the link:{neo4j-docs-base-uri}/operations-manual/4.4/tools/neo4j-admin/push-to-cloud/[Neo4j Admin push-to-cloud documentation].
```

Comment (AlexicaWright) [reply modules/ROOT/partials/import-database.adoc:36]: Sorry, I missed this one before. Again, the tool itself is not versioned independently of Neo4j, so we should try not to talk about the version of the tool, but of Neo4j.

**Comment by nvitucci** [comment]
Superseded by https://github.com/neo4j/docs-aura/pull/339.

**Pull request closed without merging**